### PR TITLE
Don't install graphviz on CI services

### PR DESCRIPTION
**Pull request by astrofrog**
I **think** graphviz isn't needed anymore as this was needed for the automodapi tests which are now in the sphinx-automodapi repo. This will speed up the CI a fair bit especially on Mac.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,25 +10,18 @@ os:
...
     - PYTHON_VERSION=3.5
     - PYTHON_VERSION=3.6 SETUPTOOLS_VERSION=dev DEBUG=True
-      CONDA_DEPENDENCIES='sphinx cython numpy packaging six appdirs'
+      CONDA_DEPENDENCIES='sphinx cython numpy packaging six appdirs coverage pytest-cov'
       EVENT_TYPE='push pull_request cron'
 
   global:
-    - CONDA_DEPENDENCIES="setuptools sphinx cython numpy"
-    - PIP_DEPENDENCIES="coveralls pytest-cov"
+    - CONDA_DEPENDENCIES="setuptools sphinx cython numpy coverage pytest-cov"
```
Comment (bsipocz) [inline .travis.yml]: It's probably better to install these in ci-helpers when ``--cov`` is stated, isn't it?

Comment (astrofrog) [reply .travis.yml]: @bsipocz - yes, though let's do that in a separate PR?

Comment (bsipocz) [reply .travis.yml]: yes, and only once it's added to ci-helpers.

**Comment by astrofrog** [comment]
Ah no apparently we still need graphviz for the tests. Maybe what I'll do is disable the inheritance graph. We really don't need to test that here.

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [reply .travis.yml]: Hmm, ci-helpers currently install pytest-cov when ``--cov`` is used. I wonder whether coverage is indeed needed or will be installed as the pytest-cov dependency anyway?

**Comment by astrofrog** [comment]
The Mac builds have now gone from ~12 to ~4 minutes

**Comment by bsipocz** [comment]
Thank you @astrofrog!

**Pull request merged**